heron/cli/src/python/submit.py

```diff
@@ -89,7 +89,7 @@ def launch_a_topology(cl_args, tmp_dir, topology_file, topology_defn_file):
       "--release_file", release_yaml_file,
       "--topology_package", topology_pkg_path,
       "--topology_defn", topology_defn_file,
-      "--topology_jar", topology_file
+      "--topology_jar", topology_file   # pex file if pex specified
```

Comment:
agree. ``--topology_jar`` definitely confuses people who submit Python topology. ``--topology_binary`` sounds good to me, or we can add ``--topology_pex`` option and group it with ``--topology_jar`` option into a [mutually exclusive group](https://docs.python.org/2.7/library/argparse.html#mutual-exclusion)